examples/Makefile

```diff
@@ -19,8 +19,8 @@
 
 all:
 	cd SingleNeuron_HH1952; make -B
-	cd Network_HodgkinHuxleyNeurons; make -B
-	cd SingleNeuron_InjectedCurrent; make -B
-	cd IntegrateAndFire_S1967; make -B
-	cd SingleNeuron_HH1952_Stochastic; make -B
+	cd ../Network_HodgkinHuxleyNeurons; make -B
+	cd ../SingleNeuron_InjectedCurrent; make -B
+	cd ../IntegrateAndFire_S1967; make -B
+	cd ../SingleNeuron_HH1952_Stochastic; make -B
```

Comment:
That was just to check whether everything is working fine and all the
examples have been correctly edited.

On Wed, Jul 15, 2015 at 3:50 PM, Pranav Kulkarni <notifications@github.com>
wrote:

> In examples/Makefile
> <https://github.com/insilico-lib/insilico/pull/26#discussion_r34664964>:
>
> > @@ -19,8 +19,8 @@
> >
> >  all:
> >  	cd SingleNeuron_HH1952; make -B
> > -	cd Network_HodgkinHuxleyNeurons; make -B
> > -	cd SingleNeuron_InjectedCurrent; make -B
> > -	cd IntegrateAndFire_S1967; make -B
> > -	cd SingleNeuron_HH1952_Stochastic; make -B
> > +	cd ../Network_HodgkinHuxleyNeurons; make -B
> > +	cd ../SingleNeuron_InjectedCurrent; make -B
> > +	cd ../IntegrateAndFire_S1967; make -B
> > +	cd ../SingleNeuron_HH1952_Stochastic; make -B
>
> Where are you running make from for examples directory?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/insilico-lib/insilico/pull/26/files#r34664964>.
>